pkg/webhook/resources/upgrade/validator.go

```diff
@@ -282,7 +283,8 @@ func (v *upgradeValidator) checkDiskSpace(node *corev1.Node, minFreeDiskSpace ui
 	if *summary.Node.Fs.AvailableBytes < minFreeDiskSpace {
 		min := units.BytesSize(float64(minFreeDiskSpace))
 		avail := units.BytesSize(float64(*summary.Node.Fs.AvailableBytes))
-		return werror.NewBadRequest(fmt.Sprintf("Node %q has insufficient free space (%s). An upgrade requires at least %s of free space.", node.Name, avail, min))
+		return werror.NewBadRequest(fmt.Sprintf("Node %q has insufficient free system disk space %s (%s). An upgrade requires at least %s of free system disk space.",
```

Comment:
`free system disk space` or `free system partition space` ?  seems `partition` will be better. We can change code and doc to use `system partition`.   @bk201  WDYT ? thanks.

@WebberHuang1118 
`An upgrade requires at least %s of free system disk space.` -> `The upgrade requires at least %s of free system disk space on each node.`  seems better.

Reply:
Both work. But partition is more accurate. The goal is to display the `/usr/local` info.

Reply:
OK, lets use `free system partition space`, I will also update doc PR.

Reply:
Updated error message according to our discussion